Makefile

```diff
@@ -0,0 +1,18 @@
+# This Makefile generates SPIR-V shaders from GLSL shaders in the examples.
+
```

Comment:
Perhaps a `.PHONY`?

Reply:
I considered this, but it is very unlikely that a file named `default` or `clean` will be created, and (in my opinion) not worth the extra complexity.

Reply:
It's just easier to have all phony targets in `.PHONY` without thinking of whether they can accidentally match the files

Reply:
Sure thing, I added them!